test/browser_tests/page_components/footer.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+function Footer() {
+  this.get = function() {
+    browser.get( '/' );
+  };
+
+  this.footer = element( by.css( '.footer' ) );
+  this.navList = this.footer.element( by.css( '.footer_nav-list' ) );
+  this.navLinks = element.all( by.css( '.footer-middle-left .footer_list a, .footer-middle-right .footer_list a' ) );
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/d59e16dd6fa8b0ad43e83530dea0a38410d2f97b